common-utils/src/main/java/com/stnetix/ariaddna/commonutils/DTO/vufs/ChunkFile.java

```diff
@@ -0,0 +1,27 @@
+package com.stnetix.ariaddna.commonutils.DTO.vufs;
+
+import java.io.Serializable;
+
+/**
+ * Describe the chunk of file
+ */
+public class ChunkFile implements Serializable {
+    private Long id;
+    private String url;
+
```

Comment:
I am not sure that this place is the best for the following comment, but...
1. I saw everything describing the _chunk_ (VUFSObject, VirtualFile), but I did not see the field which is responsible for storing data itself. I guess this could be a byte array: byte[] rawData for example. And it can be the field of VurtualFile class or ChunkFile class.